### DISCUSSION TITLE
ESP-01S Relay Board Update code

### DISCUSSION THREAD

**Discussion by SheYu09** [RavenSystem/esp-homekit-devices#2154] [MEPLHAA Scripts]
My old code is using this, I try to convert to new code but not working.

{"c":{"l":1,"n":"IT_01s_Switch"},"a":[{"0":{"r":[{"g":0,"v":1}]},"1":{"r":[{"g":0,"v":0}]},"t":1,"s":0,"b":[{"g":2,"p":1},{"g":2,"t":0}]}]}

**Comment by Gandalf8266** [comment]
I have also problems with my ESP-01S Relay board. My Relay is "on" after booting the device.
In your cases, you forgot the new "io" configuration with is necessary since HAA V12 Merlin, 
See Documentation:
https://github.com/RavenSystem/esp-homekit-devices/wiki/gpios-configuration
